docs/install-kyma-dashboard-manually.md

```diff
@@ -35,3 +35,24 @@ To run Kyma Dashboard manually on a cluster, you must install it together with I
 
 3. Wait a minute for the Dashboard to start.
 4. The Dashboard is available under your domain (for example, `https://busola.local.kyma.dev/`).
+
+## Enable TLS in the backend service
+
+By default, the communication with the backend pod is plain HTTP. Even when you use istio service mesh, the last bit of communication
```

Comment:
```suggestion
By default, the communication with the backend Pod is plain HTTP. Even when you use Istio Service Mesh, the last bit of communication between the Istio sidecar and the backend service stays plain HTTP.
```

Capitalization as in [Kyma style guide](https://kyma-project.io/community/guidelines/content-guidelines/02-style-and-terminology#capitalized-terms).
Let's not split a sentence in two lines.